doc/source/reference/distutils.rst

```diff
@@ -1,5 +1,7 @@
 .. for doctests
    >>> with open('foo.c', 'w') as f: pass
+   >>> import numpy.distutils.misc_util
+   >>> config = np.distutils.misc_util.Configuration(None, '', '.')
```

Comment:
Can you move this block down to right above the example that needs it? I don't think putting this at the top adds clarity.